src/core/production_factor.jl

```diff
@@ -304,12 +304,12 @@ function get_production_factor(wind::Wind, latitude::Real, longitude::Real, time
         @ccall hdl.ssc_data_free(data::Ptr{Cvoid})::Cvoid
 
     catch e
-        @error "Problem calling SAM C library!"
+        throw(@error("Problem calling SAM C library!"))
         showerror(stdout, e)
```

Comment:
Good question. Since we are throwing the SAM error, we probably dont get to `showerror()` anymore. I can test this.

Reply:
It appears that we could eliminate the `showerror` function call since `throw(@error...` will capture the error + the SAM error (e) and return that in results dictionary. The code wont proceed to `showerror` function call.

Reply:
Addressed in https://github.com/NREL/REopt.jl/commit/47f48ce5f097f9d10d9eba220dfd1bf4b9010e27 and https://github.com/NREL/REopt.jl/commit/69c336f9d31bc819ebef99af40899cee9accfd89